Console/retro/ConsoleWindow.cc

```diff
@@ -154,3 +161,68 @@ char ConsoleWindow::WaitNextChar()
 
     return event.message & charCodeMask;
 }
+
+// Wrapper for the GetNextEvent() function
+bool getNextEventWrapper(EventRecord *event)
```

Comment:
These functions should be static or inside namespace{} so that they won't interfere with other functions of the same name in the main program.